src/Service.Tests/SqlTests/GraphQLMutationTests/MultipleCreateMutationTests/MultipleCreateMutationTestBase.cs

```diff
@@ -0,0 +1,566 @@
+                                                { birthdate: ""2000-01-02"", name: ""Author #2"", royalty_percentage: 50.0 }
+                                              ]
+                                            }
+                                          ) {
+                                            id
+                                            title
+                                            publishers {
+                                              id
+                                              name
+                                            }
+                                            reviews {
+                                              items {
+                                                book_id
+                                                id
+                                                content
+                                                website_users {
+                                                  id
+                                                  username
+                                                }
+                                              }
+                                            }
+                                            authors {
+                                              items {
+                                                id
+                                                name
+                                                birthdate
+                                              }
+                                            }
+                                          }
+                                        }";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+            SqlTestHelper.PerformTestEqualJsonStrings(expectedResponse, actual.ToString());
+        }
+
+        /// <summary>
+        /// <code>Do : </code> Many type multiple create mutation request with entities related through 1:1, N:1, 1:N and M:N relationships, all in a single mutation request.This also a
+        /// combination relationships defined at the database layer and through the config file.
+        /// 1. a) 1:1 relationship between Review - WebsiteUser entity is defined through the config file. b) Other relationships are defined through FK constraints 
+        /// 2. Depth of this create mutation request = 2. Book --> Review --> WebsiteUser.
+        /// <code>Check : </code> Records are successfully created in all the related entities. The created items are related as intended in the mutation request.
+        /// Correct linking of the newly created items are validated by querying all the relationship fields in the selection set and validating it against the expected response.
+        /// </summary>
+        public async Task ManyTypeMultipleCreateMutationOperation(string expectedResponse)
```

Comment:
Basically, validating values for non-relationship fields like royalty_percentage since the JOIN condition during querying will not involve the non-relationship fields in the linking table.

Reply:
Good point. I suppose, I can execute a database query against the linking table and validate the records created. Will add this validation too.

Reply:
I've added linking table validations.